lib/pages/team_info_page.dart

```diff
@@ -139,13 +163,26 @@ class _TeamInfoForm extends State<TeamInfoPage> {
     return null;
   }
 
+  /// Returns error message ([String]) in case [value] is empty
+  String? _validateDriverStation(value) {
+    if (value == null) {
+      return 'Please select a driver station!';
+    }
+
+    return null;
+  }
+
   /// Extract team number from "<Team name> #<Team number>"
   int _extractTeamNumber(String fullTeamText) =>
       int.parse(fullTeamText.split("#")[1]);
 
   /// Validate form and submit it (Moving to the next page [MatchPage])
   void _submitForm() {
     if (_formKey.currentState!.validate()) {
+      if (_selectedTeamController.value == null) {
+        return;
+      }
+
       _match.number = int.parse(_matchNumberController.text);
       _match.alliance = _selectedAlliance;
       _match.team = Team()
```

Comment:
When you don't select a team it throws a ```Null check operator used on a null value``` error. Doesn't impact the app itself, so it's optional. I'd put this after checking that the user selected a value, but again - it's optional.

Reply:
The form validation ignores the team selection, something in the package is messed up. So even not selecting any team, the validation will succeed. All this does is prevent users from moving on without a team selected. No error is raised.